frontend/src/components/regions/UsersAndGroups/UsersAndGroups.test.js

```diff
@@ -72,3 +73,42 @@ test("cleanAndFilterTeams can deal with multiple words in any order", () => {
   const names = ret.map((group) => group.name);
   expect(names).toEqual(["demo group 1 AAA", "demo group 2 aaa"]);
 });
+
+test("getPrStatus should return error when pr age is older than PR_ERROR_AGE_THRESHOLD", () => {
+  const oldDate = "2021-10-13T04:45:01Z";
```

Comment:
These tests are still brittle. The tests should pass if I set PR_ERROR_AGE_THRESHOLD to 3000. 

And if IU run the tests today and they pass, then they should also pass a week from now. They should also pass a year from now.